src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -191,7 +191,7 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
                         var builder = ImmutableArray.CreateBuilder<TaggedText>();
                         builder.AddText(FeaturesResources.Awaited_task_returns_0.Substring(0, symbolIndex));
                         builder.AddRange(mainDescriptionTaggedParts);
-                        builder.AddText(FeaturesResources.Awaited_task_returns_0.Substring(symbolIndex + defaultSymbol.Length));
+                        builder.AddText(FeaturesResources.Awaited_task_returns_0[(symbolIndex + defaultSymbol).Length..]);
```

Comment:
Those parentheses are not right. Bug in the fixer, or manual change?

Reply:
manual.  good catch.  =-0   

we've fixed this in the fixer.  but i'm running an older version.